web/client/components/mapcontrols/locate/LocateBtn.jsx

```diff
@@ -94,9 +95,9 @@ class LocateBtn extends React.Component {
     };
 
     addTooltip = (btn) => {
-        let tooltip = <Tooltip id="locate-tooltip">{this.props.tooltip}</Tooltip>;
+        const tooltip = <Tooltip id="locate-tooltip"><Message msgId={this.props.tooltip} /></Tooltip>;
         return (
-            <OverlayTrigger placement={this.props.tooltipPlace} key={"overlay-trigger." + this.props.id} overlay={tooltip}>
+            <OverlayTrigger placement={this.props.tooltipPlace} key={`{overlay-trigger.${this.props.id}-${this.props.tooltip}}`} overlay={tooltip}>
```

Comment:
See [this](https://github.com/geosolutions-it/MapStore2/pull/4933/files/968bb078066b3fe3ec933cba3fd273874a3849c6#diff-b10402e060bd491419f4ec54c1eddbb0) for reference.